daemon/config_unix.go

```diff
@@ -90,6 +91,7 @@ func (config *Config) InstallFlags(cmd *flag.FlagSet, usageFn func(string) strin
 	cmd.BoolVar(&config.EnableSecrets, []string{"-enable-secrets"}, true, usageFn("Enable Secrets"))
 	cmd.StringVar(&config.CgroupParent, []string{"-cgroup-parent"}, "", usageFn("Set parent cgroup for all containers"))
 	cmd.StringVar(&config.RemappedRoot, []string{"-userns-remap"}, "", usageFn("User/Group setting for user namespaces"))
+	cmd.BoolVar(&config.RemapGroupAdds, []string{"-userns-remap-group-adds"}, false, usageFn("Add user namespace gid mapping for group-adds"))
```

Comment:
prefixing with `#` should directly hide the flag in the --help. But sure, you can also comment that's disabled.